app/src/full/java/io/homeassistant/companion/android/sensors/LocationSensorManager.kt

```diff
@@ -540,6 +552,28 @@ class LocationSensorManager : LocationSensorManagerBase() {
         ).toBoolean()
     }
 
+    private suspend fun setupSendLocationAsSetting() {
+        val supported = integrationUseCase.isHomeAssistantVersionAtLeast(2022, 2, 0)
```

Comment:
this release was 8 months ago, do we really need to check this still?

Reply:
Why not? It prevents potential reports of it not working and isn't that much effort as we need to get/create the setting anyway.

Reply:
Ok do you think its better to hide the setting in this case then? Also we should probably update the sensor description to mention this new setting too.

Reply:
I don't think we can hide settings - do you mean not creating the setting when on <2022.2?

Reply:
yea thats what I meant 😂

Reply:
Updated setting to only be added when on >= 2022.2, and added a short description of the setting (+ now the setting name actually matches the documentation).

Like you mentioned most users will probably be on a supported version, so I think it is fine that the description always mentions the version even though a few users don't have the option.